src/main/java/org/medicmobile/webapp/mobile/EmbeddedBrowserActivity.java

```diff
@@ -66,6 +70,7 @@ public void onReceiveValue(String result) {
 
 
 //> ACTIVITY LIFECYCLE METHODS
+	@SuppressLint("ClickableViewAccessibility")
```

Comment:
Because the onTouchEvent() isn't calling View#performClick

Reply:
Because we don't want to click: https://developer.android.com/reference/android/view/View#onTouchEvent(android.view.MotionEvent)